components/dashboard/components/TimelineChart.jsx

```diff
@@ -24,7 +24,18 @@ const drawGantt = (props) => {
     // create map of id: name
     // local user will always be first.
     logger.debug('sorted participants:', participants);
-    const participantNames = participants.map((p) => p.name);
+
+    //since the decision was made to limit the length of the name on the y-axis if overflowing,
+    //the breakpoint at which the name should be truncated,
+    //is 720px (width of parent, <MeetingViz/>)
+    //what seemed to work was truncating the name to 13 characters, plus an ellipsis
+
+    const truncateBreakPoint = 720;
+    const minTruncateLength = 13;
```

Comment:
What is a minimum truncate length vs a maximum truncate length?
And what is a truncate breakpoint. Using the const's is great but I'm still having a little trouble interpreting your comment (which if it answered my questions when I read the names would be better).
The truncateBreakPoint actually seems to be related to a width of something (is props.width the pixel width of the chart?) and is used to select between the minTruncateLength and the maxTruncateLength.
Those seem to be maxNameLenWide and maxNameLenNarrow or some such, not max and min. The breakpoint is really something like maxNarrowWidth (or something like that, I don't love Wide, Narrow as descriptions but they seem close)

Reply:
Alright, I've made those adjustments.

Reply:
Thanks, does the reasoning behind why I was requesting changes make sense to you?

Reply:
I mean, I think the reason the discussion is happening is because the solution isn't ideal in the first place, and not having descriptive constants or comments made the solution even more ambiguous. Since we have the width of the parent, it would be ideal to truncate it dynamically...as you mentioned. But I think this is a decent enough solution, haha seems fair to me!